ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+---
+# tasks file for cyhy_feeds
+- name: Grab GitHub OAuth token from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: github_oauth_token.yml
+    dest: /tmp/github_oauth_token.yml
+    mode: get
+  become: no
+
+- name: Load GitHub OAuth token from YML file
+  local_action:
+    module: include_vars
+    file: /tmp/github_oauth_token.yml
+  become: no
+
+- name: Grab mongo users from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: mongo_users.yml
+    dest: /tmp/mongo_users.yml
+    mode: get
+  become: no
+
+- name: Load mongo users token from YML file
+  local_action:
+    module: include_vars
+    file: /tmp/mongo_users.yml
+    name: mongo_users
+  become: no
+
+- name: Create the /var/cyhy/scripts/cyhy-feeds/
+  file:
+    path: /var/cyhy/scripts/cyhy-feeds/cyhy_extracts
+    state: directory
+
+- name: Create /etc/cyhy dir
+  file:
+    path: /etc/cyhy
+    state: directory
+
+- name: Download cyhy extract from GitHub
+  get_url:
+    url: "https://raw.githubusercontent.com/jsf9k/cyhy-feeds/develop/aws_jobs/cyhy-data-extract.py"
+    headers:
+      Authorization: "token {{ github_oauth_token }}"
+    dest: /var/cyhy/scripts/cyhy-feeds/cyhy-data-extract.py
+    mode: 0755
+
+<<<<<<< HEAD
+- name: Download dmarc extract from GitHub
+  get_url:
+    url: "https://raw.githubusercontent.com/jsf9k/cyhy-feeds/develop/aws_jobs/dmarc.py"
+    headers:
+      Authorization: "token {{ github_oauth_token }}"
+    dest: /var/cyhy/scripts/cyhy-feeds/dmarc.py
+    mode: 0755
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+- name: Grab cyhy extract config file from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: cyhy-data-extract.cfg
+    dest: /tmp/cyhy-data-extract.cfg
+    mode: get
+  become: no
+
+- name: Grab gpg public key from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: ncats_lab_public_gpg.key
+    dest: /tmp/ncats_lab_public_gpg.key
+    mode: get
+  become: no
+
+<<<<<<< HEAD
+- name: Grab cyhy private key from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: ncats_lab_private_gpg.key
+    dest: /tmp/ncats_lab_private_gpg.key
+    mode: get
+  become: no
+
+- name: Grab cyhy johns public key from S3
```

Comment:
I would consider renaming this public key (both in the bucket and at the destination) to reflect the function of the key, rather than the name of the person who generated it, i.e. something like `nsd_cyhy_daily_extract_public.key`